playbooks/service-provision.yaml

```diff
@@ -109,6 +109,27 @@
          Check agnosticv-operator logs:
            oc logs -n agnosticv-operator deployment/agnosticv-operator
 
+  - name: Check if user {{ cloudforms_username }} exists
+    k8s_info:
+      kubeconfig: "{{ kubeconfig }}"
+      api_version: v1
+      kind: User
+      name: "{{ cloudforms_username }}"
+    register: r_user
+
+  - name: Create User {{ cloudforms_username }}
+    when: r_user.resources | length == 0
+    k8s:
+      kubeconfig: "{{ kubeconfig }}"
+      definition:
+        apiVersion: v1
```

Comment:
Needs to be `apiVersion: user.openshift.io/v1`

Reply:
I've tested using only `kind: User` and works, should I force to use `apiVersion: user.openshift.io/v1`